scheduler/plugin/plugin_test.go

```diff
@@ -1350,6 +1352,52 @@ func TestPostBind(t *testing.T) {
 	}
 }
 
+// This test checks whether framework.handle.EventRecorder.Eventf can be called within wasm file.
+func TestEventf(t *testing.T) {
+	tests := []struct {
+		name        string
+		guestURL    string
+		pod         *v1.Pod
+		expectedMsg string
+	}{
+		{
+			name:        "call plugin.Set",
+			guestURL:    test.URLExampleNodeNumber,
+			pod:         test.PodSmall,
+			expectedMsg: "PreScore not match lastNumber Skip ",
+		},
+		{
+			name:        "call each plugin's Set",
+			guestURL:    test.URLExampleAdvanced,
+			pod:         test.PodSmall,
+			expectedMsg: "PreScore not match lastNumber Skip ",
+		},
```

Comment:
What're those test case titles meaning? (what's plugin.Set or plugin's Set they're referring to?)

Reply:
Sorry, these are not clear. Changed titles.
Does it make sense?

Reply:
yep